frontend/src/app/my-links/page.tsx

```
@@ -0,0 +1,196 @@
+
+        {/* Pinned Folders Top Section - Strictly 5 Columns Desktop */}
+        {pinnedFolders.length > 0 && (
+          <div className="mb-5">
+            <h2 className="text-xs font-bold text-gray-800 dark:text-gray-100 mb-3 ml-1">Pinned</h2>
+            <div className="grid grid-cols-2 lg:grid-cols-5 gap-2">
+              {pinnedFolders.map((folder, idx) => (
+                <div
+                  key={folder.memberFolderId}
+                  onClick={() => setSelectedFolderId(folder.memberFolderId)}
+                  className={`${PINNED_COLORS[idx % PINNED_COLORS.length]} text-white rounded-lg p-2.5 flex flex-col justify-between h-20 shadow-sm relative overflow-hidden group hover:scale-[1.02] transition-transform cursor-pointer`}
+                >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)